components/cache/introduction.rst

```diff
@@ -0,0 +1,73 @@
+.. index::
+   single: Cache
+   single: Performance
+   single: Components; Cache
+
+The Cache Component
+===================
+
+    The Cache component provides a strict `PSR-6`_ implementation for adding
+    cache to your applications. It is designed to have a low overhead and it
+    ships with ready to use adapters for the most common caching backends.
+
+Installation
+------------
+
+You can install the component in 2 different ways:
+
+* :doc:`Install it via Composer </components/using_components>` (``symfony/cache`` on `Packagist`_);
+* Use the official Git repository (https://github.com/symfony/cache).
+
+.. note::
+
+    In Symfony applications this component is integrated (and enabled by
+    default) through the FrameworkBundle.
+
+Key Concepts
+------------
+
+Before starting to use the Cache component, it's important that you learn the
+meaning of some key concepts:
+
+* **Item**, a single unit of information stored as a key/value pair, where the
+  key is the unique identifier of the information and the value is its contents;
+* **Pool**, a logical repository of cache items. All cache operations (saving
+  items, looking for items, etc.) are performed through the pool. Applications
+  can define as many pools as needed.
+* **Adapter**, it implements the actual caching mechanism to store the
+  information in the filesystem, in a database, etc. The component provides
+  several ready to use adapters for common caching backends (Redis, APCu, etc.)
```

Comment:
Oh wait... Here we're not talking about the "pools" concept introduced in the FrameworkBundle. Overloaded terms are far too confusing...